ios/MaasCore/Sources/MaasComponents/Components/BottomSheet/BottomSheetEnvironment.swift

```diff
@@ -0,0 +1,35 @@
+import Combine
+import SwiftUI
+
+public class BottomSheetEnvironment: ObservableObject {
+
+    static var animationDuration: Double = 0.3
```

Comment:
Would suggest to keep animationType shared as well
```suggestion
    static var defaultAnimation = Animation.easeInOut(duration: 0,3)
```